.babelrc

```diff
@@ -1,9 +1,34 @@
 {
-  "presets": ["es2015"],
-  "plugins": ["transform-decorators-legacy", "transform-class-properties", "transform-runtime"],
+  "presets": [
+    [
+      "@babel/preset-env", 
+      {
+        "targets": {
+          "node": "8.11",
+          "browsers": [
```

Comment:
Might be worth tapping into `browserlist` instead, and telling it to target `current - 2` rather than hardcoding the names and versions of only four browsers?

Reply:
No, because `last 2 version` is non deterministic, i.e., since the browser versions changes at each browser new release the build output could change from time to time.

Reply:
Which is what you'd want: it's why build output gets versioned so that you can note the support matrix in the changelog?